presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -384,6 +385,19 @@ public TaskManagerConfig setMaxPagePartitioningBufferSize(DataSize size)
         return this;
     }
 
+    public int getMaxPagePartitioningBufferCount()
+    {
+        return maxPagePartitioningBufferCount;
+    }
+
+    @Config("driver.max-page-partitioning-buffer-count")
```

Comment:
`@Min(1)`

Reply:
Thanks @tdcmeehan ! I updated the PR with the following
```
    @Min(1)
    public int getMaxPagePartitioningBufferCount()
    {
        return maxPagePartitioningBufferCount;
    }
```